assets/js/components/BadgeWithTooltip.stories.js

```diff
@@ -19,23 +19,24 @@
 /**
  * Internal dependencies
  */
-import PartialDataBadge from './PartialDataBadge';
+import BadgeWithTooltip from './BadgeWithTooltip';
 
 function Template( args ) {
-	return <PartialDataBadge { ...args } />;
+	return <BadgeWithTooltip { ...args } />;
 }
 
 export const Default = Template.bind( {} );
 Default.storyName = 'Default';
 
 Default.args = {
+	label: 'Title for badge with tooltip',
 	tooltipTitle:
 		'Still collecting full data for this timeframe, partial data is displayed for this group',
```

Comment:
```suggestion
		'This is an example of tooltip content for a badge with tooltip',
```

This is a small detail.